tutorials/math/interpolation.rst

```diff
@@ -109,7 +109,11 @@ And again, it will produce the following motion:
 Smoothing motion
 ----------------
 
-Interpolation can be used to smooth movement, rotation, etc. Here is an example of a circle following the mouse using smoothed motion:
+Interpolation can be used to smoothly follow a moving target value, such as a
+position or a rotation. Each frame, ``lerp()`` moves the current value towards
+the target value by a fixed percentage of the remaining difference between the values.
```

Comment:
"Percentage" is technically incorrect here, since the value is in the range of `[0,1]` rather than `[0,100]`. "Portion" or maybe "fraction" is more technically correct, but have the wrong connotations - in English, "portion" can also mean a fixed amount rather, rather than a value that depends on the total value.

Reply:
I think this is fine as is, I agree with the reasoning.

Reply:
I'd be happy to do a pass on the rest of the page too, but I think that's slightly out of scope of this PR which just aims to improve the section on using lerp() to follow a moving target (or "lerp smoothing").

Reply:
In that case I'm fine with this as is, too. That example is better for the beginning rather halfway down the page, anyway.

Reply:
I made a change to the very first paragraph:
- bringing the removed sentence to the beginning of the page
- removing one of the banned words "basic"
- changing "graphics developer" to "game developer"

To be clear, the rest of this page still needs more work, but it's beyond the scope of this PR.